man-roxygen/param_learner_properties.R

```diff
@@ -4,10 +4,12 @@
 #'   The following properties are currently standardized and understood by learners in \CRANpkg{mlr3}:
 #'   * `"missings"`: The learner can handle missing values in the data.
 #'   * `"weights"`: The learner supports observation weights.
+#'   * `"offset"`: The learner can incorporate offset values to adjust predictions.
 #'   * `"importance"`: The learner supports extraction of importance scores, i.e. comes with an `$importance()` extractor function (see section on optional extractors in [Learner]).
```

Comment:
@be-marc here `hotstart_forward`, `hotstart_backward` and `featureless` are not documented, but they should be perhaps?

Reply:
Thanks did that in main